src/Common/tests/Resources/Strings.resx

```diff
@@ -134,4 +153,7 @@
   <data name="IO_SharingViolation_File" xml:space="preserve">
     <value>IO_SharingViolation_File {0}</value>
   </data>
+  <data name="IO_PathTooLong_Path" xml:space="preserve">
```

Comment:
I can't see coreclr (on phone) but hopefully the same strings are used in these changes: I rewrote some when I did the coreclr change, to be more helpful

Reply:
Yes. Actually, I took the string from coreclr.